src/components/dropdown_styles.css

```diff
@@ -0,0 +1,148 @@
+/* Dropdown Button */
+.dropbtn {
+display: fixed;
+flex-direction: row;
+justify-content: center;
+align-items: center;
+padding: 0px;
+gap: 8px;
+width: 98px;
+height: 40px;
+background: #006D7D;
+border-radius: 8px;
+flex: none;
+order: 0;
+align-self: flex-start;
+flex-grow: 0;
+margin-left: 100px;
```

Comment:
Lets remove these margin-left and margin-top. This will make it easier to style with the other dropdowns later!